src/figuresClasses.ts

```diff
@@ -1,11 +1,84 @@
-export interface Figure {}
+export interface Figure {
+  getArea(): number;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape: string = 'triangle';
+
+  public color: string;
+
+  constructor(
+    color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
+
+    const sides = [a, b, c].sort((x, y) => x - y);
+
+    if (sides[2] >= sides[0] + sides[1]) {
+      throw new Error(`Sides ${a}, ${b} and ${c} can't form a triangle`);
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. In this case, your error message is quite descriptive, but it's better to provide the user with more context about why these particular side lengths can't form a triangle. For example, you could say 'The sum of the lengths of the two shorter sides must be greater than the length of the longest side for a valid triangle'.